linfa-ensemble/src/random_forest/hyperparameters.rs

```diff
@@ -0,0 +1,80 @@
+use linfa_trees::DecisionTreeParams;
+
+#[derive(Clone, Copy)]
+pub struct RandomForestParams {
+    // number of decision trees to fit
+    pub n_estimators: usize,
+    // single tree hyperparameters
+    pub tree_hyperparameters: DecisionTreeParams,
+    // max number of features for the ensemble
+    pub max_features: u64,
+    // use all dataset or not
+    pub bootstrap: bool,
```

Comment:
I think this doc is slightly misleading / also the flag is unused at the moment.